deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -92,26 +92,41 @@ spec:
           - kind: ConfigMaps # operator-sdk uses this for leader election
             version: v1
         specDescriptors:
+          - description: 'How many pod instances should be members of the cluster'
```

Comment:
why is this a question.. recommend rewording

Reply:
good suggestion. thanks.